src/game/types/index.ts

```diff
@@ -344,6 +345,10 @@ export interface ShellNotificationPayload {
   [ShellNotificationType.TOOL_CARD_PLAYED]: {
     toolCard: ToolInstance
   }
+
+  [ShellNotificationType.GENERIC_INFO]: {
+    message: string
+  }
```

Comment:
The message content should be handled in `src/ui/components/Match/useSnackbar.ts`. In this case, I think this payload can be left empty:
```suggestion
  [ShellNotificationType.ALL_CROPS_WATERED]: {}
```